client/components/Menu/Menu.jsx

```diff
@@ -91,11 +69,11 @@ class ResponsiveDrawer extends React.Component {
           {['Forum', 'Creators', 'Contact', 'Setting'].map((text, index) => (
             <ListItem button key={text}>
 
-            {/* Original code */}
-            {/* <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon> */}
-            {/* Need to figure out how to correctly add icon */}
+              {/* Original code */}
+              {/* <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon> */}
+              {/* Need to figure out how to correctly add icon */}
 
-              <ListItemIcon>{index % 2 === 0 ? <Add/> : <Face />}</ListItemIcon>
+              <ListItemIcon>{index % 2 === 0 ? <Add /> : <Face />}</ListItemIcon>
```

Comment:
index % 2 === 0 is the same as !(index % 2)
Or in other words:
`index % 2 ? <Face /> : <Add />`

Reply:
Right! This was based on the basic response drawer code with using material-ui icons. I was messing with it a little bit, but I think that code will have to be changed essentially.